config.go

```diff
@@ -251,3 +273,7 @@ func utcISO8601TimeEncoder(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
 func utcEpochTimeEncoder(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
 	enc.AppendInt64(t.UTC().UnixNano())
 }
+
+func localTimeEncoder(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
+	enc.AppendString(t.Local().Format("2006-01-02 15:04:05.000"))
```

Comment:
Also, ISO8601 encode format should be used here to be in sync with the current encoder.